config.default

```diff
@@ -489,53 +489,146 @@ colorbarLevelsDifference = [-150, -80, -30, -10, 0, 10, 30, 80, 150]
 # Nov, Dec, JFM, AMJ, JAS, OND, ANN)
 comparisonTimes =  ['JFM', 'JAS', 'ANN']
 
-[climatologyMapSeaIceConcThick]
+[climatologyMapSeaIceConcNH]
 ## options related to plotting horizontally remapped climatologies of
-## sea ice concentration and thickness against reference model results and
-## observations
+## sea ice concentration against reference model results and observations
+## in the northern hemisphere (NH)
+
+# colormap for model/observations
+colormapNameResult = inferno
+# color indices into colormapName for filled contours
+colormapIndicesResult = [20, 80, 110, 140, 170, 200, 230, 255]
+# colormap levels/values for contour boundaries
+colorbarLevelsResult = [0.15, 0.3, 0.5, 0.7, 0.8, 0.85, 0.9, 0.95, 1]
```

Comment:
we used to have to different colorbarLevels for winter and summer, but I guess it is OK to make them the same across seasons.

Reply:
Oh, yes, I forgot to mention that and I'm glad you noticed.  I personally think it's a lot more intuitive to have the same color bar for all seasons so you can compare them, but it does depend on what your needs are.  Is there a strong reason to have them be different, other than trying to span the full range of the color bar?

Reply:
I think the reason was that, when a lot of ice is present, one wants to see more details in the high ice-coverage areas. But I have heard conflicting opinions about this, so let me re-make the plots now, and if we like them we'll just keep it like this.

Reply:
Okay, please let me know what your tests show.  I think having 16 color maps to work with (2 variables, 2 hemispheres, 2 seasons, model/obs and difference) will be a bit much for users so that was partly why I was hoping to simplify.  But I can appreciate that the ability to emphasize different ends of the color map could be useful.